README.md

```diff
@@ -37,9 +37,9 @@ An **Image Search** App to find **Anime** Details related to the Image. Created
 
 This package currently is only tested on android and the web platform.
 
-- [x] android
+- [x] [android](https://github.com/OutdatedGuy/Whats-That-Anime/releases/download/v2.0.0/android-x86_64-release.apk)
```

Comment:
Linking to only one architecture is a bad idea as all phones uses different chipsets.